libp2p/Host.h

```diff
@@ -196,6 +196,9 @@ class Host: public Worker
     /// Resets acceptor, socket, and IO service. Called by deallocator.
     void stop();
 
+    /// Stop registered capabilities, typically done when the network is being shut down.
+    void stopCapabilities();
```

Comment:
This shouldn't be public, I think

Reply:
Whoops, yes you're right, that was an accident 😄